app/src/main/java/com/termux/app/ExtraKeysView.java

```diff
@@ -40,7 +40,7 @@
     public ExtraKeysView(Context context, AttributeSet attrs) {
         super(context, attrs);
     }
-    
+
```

Comment:
You should change empty lines like that, it creates noise for the commit.
Change your editor settings so that it doesn't strip spaces at the end of lines :)

Reply:
I think it is fine to include these minor fixes in the same PR/commit, as long as it is in the same file(s) as the rest of the changes it does not create much noise

Reply:
When looking at the commit it's more difficult to find the lined that were modified (66 additions, 50 deletions, but in reality it's only like 20), some diff viewer allows to hide whitespace only modifications but I don't see that on GitHub.

But anyway my comment was a detail, not very important :)

The next action is now at [the other PR](https://github.com/trygveaa/termux-app/pull/1) 😀

Reply:
It's not my editor doing it, I removed them explicitly because they are annoying. I considered putting it in a separate commit, but since it was only whitespace, I figured it wasn't necessary. You can hide whitespace changes in GitHub by pressing the cog above the files and selecting "Hide whitespace changes".

Reply:
It's funny cause I prefer having them, and you find them annoying 😛
Haha, oh yeah, I missed the setting "Hide whitespace" I'm tired ^^'
Anyway, have you seen my commits of this night and the pr on your repo ? :)

Reply:
> The next action is now at the other PR :grinning:

Those changes are now included in this PR.